### DISCUSSION TITLE
Trouble with swapping surround plugin

### DISCUSSION THREAD

**Discussion by timjackleus** [LazyVim/LazyVim#906] [Q&A]
I'm very used to "kylechui/nvim-surround" keymappings (like tpope's surround etc) and would like to keep those mappings for surround features.
Only installing  kylechui/nvim-surround does not work due to other plugins but instead of disabling them, can I make sure nvim-surround is loaded last? or are there any other best practices?

**Comment by dpetka2001** [comment]
What doesn't work for you? I just installed it and the default keybindings work just fine and do what it's supposed to do.

**Reply by timjackleus** [reply]
Thank you for checking, commands such as ysiw, ysaw, cs"', ds] and so on are being interrupted for me, I disabled leap but there is still something in the way.

**Reply by dpetka2001** [reply]
I don't have leap disabled and it works fine with it enabled. They have different keybindings.

**Comment by timjackleus** [comment]
Turns out it I had it incorrectly set up, this is what it should look like:
```
{
    "kylechui/nvim-surround",
    version = "*", -- Use for stability; omit to use `main` branch for the latest features
    event = "VeryLazy",
    config = function()
        require("nvim-surround").setup()
    end
}
```

Thanks again for checking @dpetka2001 and sorry for wasting your time!

**Reply by dpetka2001** [reply]
Yes this is the configuration it has on the site as well. I thought you just took it from there and that's why I didn't ask about it. No worries.
